test-network-function/networking/suite.go

```diff
@@ -32,14 +36,25 @@ import (
 	"github.com/test-network-function/test-network-function/pkg/tnf"
 	"github.com/test-network-function/test-network-function/pkg/tnf/handlers/nodeport"
 	"github.com/test-network-function/test-network-function/pkg/tnf/handlers/ping"
+	"github.com/test-network-function/test-network-function/pkg/tnf/handlers/podnodename"
 	"github.com/test-network-function/test-network-function/pkg/tnf/interactive"
 	"github.com/test-network-function/test-network-function/pkg/tnf/reel"
-	"github.com/test-network-function/test-network-function/pkg/utils"
+
 	"github.com/test-network-function/test-network-function/test-network-function/results"
 )
 
 const (
-	defaultNumPings = 5
+	commandportdeclared = "oc get pod %s -n %s -o json  | jq -r '.spec.containers[%d].ports'"
+	commandportlisten   = "ss -tulwnH"
+	defaultNumPings     = 5
+	ocCommandTimeOut    = time.Second * 10
+	indexprotocolname   = 0
+	indexport           = 4
+)
+
+var (
+	declaredPort  = make(map[int]string)
+	listeningPort = make(map[int]string)
```

Comment:
Are these still needed since you changed the functions to return the values?

Reply:
no no , I was forget to delete this , I delete now.